content/docs/concepts/security/index.md

```diff
@@ -236,8 +219,50 @@ Istio provides two types of authentication:
+  Istio enables request-level authentication with JSON Web Token (JWT) validation
+  and a streamlined developer experience for [Auth0](https://auth0.com/), [Firebase Auth](https://firebase.google.com/docs/auth/),
+  [Google Auth](https://developers.google.com/identity/protocols/OpenIDConnect), and custom auth.
+
+In both cases the authentication policies are stored in the `Istio config store` via a custom Kubernetes API,
+and are kept up to date (along with keys where appropriate) for each proxy by Pilot.
+Istio also supports authentication in permissive mode to understand how a policy change would affect your security posture
+before it becomes effective.
+
+### Mutual TLS authentication
+
+Istio tunnels service-to-service communication through the client side and server side [Envoy proxies](https://envoyproxy.github.io/envoy/).
+For a client to call a server, the steps followed are:
+
+1. Istio re-routes the outbound traffic from a client to the client's local sidecar Envoy.
+
+1. The client side Envoy starts a mutual TLS handshake with the server side Envoy.
+   During the handshake, the client side Envoy also does a [secure naming](/docs/concepts/security/#secure-naming) check to verify that
+   the service account presented in the server certificate is authorized to run the target service.
+
+1. The client side Envoy and the server side Envoy establish a mutual TLS connection,
+   and Istio forwards the traffic from the client side Envoy to the server side Envoy.
+
+1. After authorization, the server side Envoy forwards the traffic to the server service through local TCP connections.
+
+#### Secure naming
+
+The secure naming information contains *N-to-N* mappings from the server identities (encoded in certificates)
+to the service names (referred by discovery service or DNS).
+A mapping from identity `A` to service name `B` means "`A` is allowed/authorized to run service `B`".
+This information is generated by Pilot (by watching the Kubernetes apiserver)
+and securely distributed to the sidecar Envoys.
+The following example explains why secure naming is critical in authentication.
+
+Suppose the legitimate servers that run the service `datastore` only use the identity `infra-team`.
+A malicious user has key/certificate with identity `test-team`.
+He intends to impersonate the service so that he could inspect the data sent from the clients.
+The malicious user deploys a forged server, with the key/certificate with identity `test-team`.
+Suppose he has successfully hacked the discovery service or DNS to map the service name `datastore` to his forged server.
+
+When a client calls the service `datastore`, it extracts the identity `test-team` from the server's certificate,
+and checks whether `test-team` is allowed to run `datastore` with the secure naming information.
+The client detects that `test-team` is **NOT** allowed to run the service `datastore`, so it fails the authentication.
```

Comment:
Do not use capital letters for emphasis, bold should do the trick.
The client detects that `test-team` is **not** allowed to run the `datastore` service and the authentication fails.